backend/apps/health/views.py

```
@@ -0,0 +1,263 @@
+            cache.set("health_check_ping", "pong", timeout=5)
+            value = cache.get("health_check_ping")
+            if value == "pong":
+                result["details"]["response_time_ms"] = round((time.time() - start) * 1000, 2)
+                # Get Redis info
+                try:
+                    redis_client = redis.from_url(settings.REDIS_URL)
+                    info = redis_client.info()
+                    result["details"]["used_memory_human"] = info.get("used_memory_human", "N/A")
+                    result["details"]["connected_clients"] = info.get("connected_clients", "N/A")
+                    result["details"]["uptime_days"] = info.get("uptime_in_days", "N/A")
+                except:
```

Ruff (0.15.20)

[error] Do not use bare `except`

(E722)